content/en/docs/tasks/security/automtls/index.md

```diff
@@ -0,0 +1,276 @@
+NAMESPACE      NAME                          AGE
+istio-system   grafana-ports-mtls-disabled   5m
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl get meshpolicies.authentication.istio.io
+NAME      AGE
+default   3m
+{{< /text >}}
+
+Last but not least, verify that there are no destination rules that apply on the example services. You can do this by checking the `host:` value of
+ existing destination rules and make sure they do not match. For example:
+
+{{< text bash >}}
+$ kubectl get destinationrules.networking.istio.io --all-namespaces -o yaml | grep "host:"
+    host: istio-policy.istio-system.svc.cluster.local
+    host: istio-telemetry.istio-system.svc.cluster.local
+{{< /text >}}
+
+You can verify setup by sending an HTTP request with `curl` from any `sleep` pod in the namespace `foo`, `mixed` or `legacy` to either `httpbin.foo`, `httpbin.mixed` or `httpbin.legacy`. All requests should succeed with HTTP code 200.
+
+For example, here is a command to check `sleep.mixed` to `httpbin.foo` reachability:
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n mixed -o jsonpath={.items..metadata.name}) -c sleep -n mixed -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+200
+{{< /text >}}
+
+This one-liner command conveniently iterates through all reachability combinations:
+
+{{< text bash >}}
+$ for from in "foo" "mixed" "legacy"; do for to in "foo" "mixed" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl "http://httpbin.${to}:8000/ip" -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
+sleep.foo to httpbin.foo: 200
+sleep.foo to httpbin.mixed: 200
+sleep.foo to httpbin.legacy: 200
+sleep.mixed to httpbin.foo: 200
+sleep.mixed to httpbin.mixed: 200
+sleep.mixed to httpbin.legacy: 200
+sleep.legacy to httpbin.foo: 200
+sleep.legacy to httpbin.mixed: 200
+sleep.legacy to httpbin.legacy: 200
+{{< /text >}}
+
+In the setup, we start with `PERMISSIVE` for all services in the mesh. Combined with automatic
+mutual TLS enabled, sleep from `foo` and `mixed` client, send mutual TLS traffic to `httpbin.foo`
+service and plain text to `httpbin.legacy` service. For `httpbin.mixed`, the client sidecar send
+mutual TLS to the pod with sidecar injected deployment, and plain text to the pod without sidecar.
+
+### Configure mutual TLS STRICT
+
+The default installation configures all the service in `PERMISSIVE` mode by default, which accepts
+both plain text and mutual TLS traffic. Now we configure `httpbin.foo` service to mutual TLS
+`STRICT` mode, only accepting mutual TLS traffic.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n foo -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "httpbin"
+spec:
+  targets:
+  - name: httpbin
+  peers:
+  - mtls: {}
+EOF
+{{< /text >}}
+
+Now the requests from the clients without sidecar start to fail.
```

Comment:
You should explicitly say which case(s) fail and exactly why (what's happening now).

The important thing to show (and explain) in this task is what is different because we're using the new auto mTLS feature.

Reply:
done. explaining the delta with auto mtls client `sleep.full` as well.